openseeds/ols-9/index.md

```diff
@@ -0,0 +1,365 @@
+---
+layout: page
+toc: true
+title: Welcome to Open Seeds Cohort 9
+image: /images/syllabus.jpg
+photos:
+  name: Niklas Morberg
+  license: CC BY-NC 2.0
+  url: https://flic.kr/p/5BXB6s
+redirect_from:
+  - /OLS-9
+  - /ols-9
+---
+
+{% include _includes/cohort-metadata.html cohort='ols-9' program='openseeds' %}
+
+# The Open Seeds 9 program
```

Comment:
It should probably be mentioned that OLS-9 is funded by DRA. @yochannah @malvikasharan  what do you think?